### PR TITLE
Playground

### PR DESCRIPTION
**Pull request by Cepeen**

### CODE REVIEW

File: lib/utils/utils.dart [hunk 1]
```diff
@@ -58,6 +58,15 @@ void checkAndExecuteOrNot(String macAddress, String ipAddress, BuildContext cont
   }
 }
 
+void checkAndExecuteOrNotNE(String macAddress, String ipAddress) async {
+  final connectivityResult = await Connectivity().checkConnectivity();
+
+  if (connectivityResult == ConnectivityResult.mobile ||
+      connectivityResult == ConnectivityResult.wifi) {
+    sendMagicPacket(macAddress, ipAddress);
+  } else {}
```
Comment (MarekJAM) [inline lib/utils/utils.dart:67]: Unnecessary else

File: lib/models/hosts.dart [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
+  });
 
-  Host(this.hostId, this.hostName, this.ipAddress, this.macAddress);
   Map<String, dynamic> toJson() => {
         'hostId': hostId,
         'hostName': hostName,
         'ipAddress': ipAddress,
         'macAddress': macAddress,
+        'time': '${pickedTime.hour}:${pickedTime.minute}',
+        'isChecked': isChecked, // Include the new property in the JSON serialization
```
Comment (MarekJAM) [inline lib/models/hosts.dart:26]: Comment seems to be leftover

File: lib/models/hosts.dart [hunk 3]
```diff
@@ -1,21 +1,40 @@
...
-        json['macAddress'],
+        hostId: json['hostId'] ?? '',
+        hostName: json['hostName'],
+        ipAddress: json['ipAddress'],
+        macAddress: json['macAddress'],
+        pickedTime: TimeOfDay(
+          hour: int.parse(json['time'].split(':')[0]),
+          minute: int.parse(json['time'].split(':')[1]),
+        ),
+        isChecked: json['isChecked'] ?? 0, // Include the new property in the JSON deserialization
```
Comment (MarekJAM) [inline lib/models/hosts.dart:38]: Comment seems to be leftover

File: lib/utils/utils.dart [hunk 4]
```diff
@@ -58,6 +58,15 @@ void checkAndExecuteOrNot(String macAddress, String ipAddress, BuildContext cont
   }
 }
 
+void checkAndExecuteOrNotNE(String macAddress, String ipAddress) async {
```
Comment (MarekJAM) [inline lib/utils/utils.dart:61]: At the moment this entire method could be removed, as nothing happens if there's no internet anyway.
I'd move it to some controller class, first check if there's internet connection (if no then display snackbar) and then call sendMagicPacket.

Comment (Cepeen) [reply lib/utils/utils.dart:61]: I agree, but  I want to keep this solution at the moment.

**Pull request closed without merging**